robotiq_s_model_articulated_gazebo_plugins/src/RobotiqHandPlugin.cpp

```diff
@@ -0,0 +1,877 @@
+/*
+ * Copyright 2014 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <robotiq_s_model_articulated_msgs/SModelRobotInput.h>
+#include <robotiq_s_model_articulated_msgs/SModelRobotOutput.h>
+#include <ros/ros.h>
+#include <string>
+#include <vector>
+#define private public 
+#include <gazebo/common/Plugin.hh>
+#include <gazebo/common/Time.hh>
+#include <gazebo/math/Angle.hh>
+#include <gazebo/physics/physics.hh>
+#include <robotiq_s_model_articulated_gazebo_plugins/RobotiqHandPlugin.h>
+#undef private
+
+// Default topic names initialization.
+const std::string RobotiqHandPlugin::DefaultLeftTopicCommand  =
+  "/left_hand/command";
```

Comment:
I'd prefer to leave default at left_hand. Someone can call the macro of the URDF with r_ and spawn a right hand and have it work out of the box.